controllers/case-controller.js

```diff
@@ -53,7 +59,11 @@ const CaseController = (Case, sequelize) => {
       order: [['createdAt', 'DESC']],
       limit,
       offset,
-      include: { association: 'connectedContacts', required: true },
+      include: {
```

Comment:
Do we definitely want to do this in a list operation - include all the contacts for every case, and all their CSAM reports? That might start producing some gnarly queries and big recordsets if we don't strictly need all that data in the list case

Reply:
Yes we are now doing poorly with this kind of screens. We should do something like "return ids and strictly needed details" for the "lists" and only fetch the entire record on "details" screens. Something to keep in mind if we refactor.